tck/src/main/java/org/eclipse/microprofile/lra/tck/TckParticipantTests.java

```diff
@@ -68,16 +69,26 @@ public void validWebApplicationExceptionReturnedTest() {
         Response response = resourcePath.request().get();
         URI lraId = URI.create(checkStatusReadAndCloseResponse(Response.Status.OK, response, resourcePath));
 
-        assertEquals("Non JAX-RS @Complete method throwing WebApplicationException shoud have been called", 
-            1, lraMetricService.getMetric(LRAMetricType.COMPLETE, lraId));
+        assertEquals("Non JAX-RS @Complete method throwing WebApplicationException shoud have been called",
+            1, lraMetricService.getMetric(LRAMetricType.Completed, lraId));
         assertEquals("@Compensate method should not have been called as LRA completed succesfully",
-            0, lraMetricService.getMetric(LRAMetricType.COMPENSATE, lraId));
+            0, lraMetricService.getMetric(LRAMetricType.Compensated, lraId));
 
+        assertEquals("@Compensate method should not have been called as LRA completed succesfully",
```

Comment:
this asserts seems to be duplicated previous one on line 74

Reply:
These asserts were not introduced by the PR

Reply:
this line (77) is indicated as added by GitHub

Reply:
No I mean the assert was already there. What changed was the capitalisation of the enum constant. But I will update the text.